docs/DeveloperGuide.md

```diff
@@ -638,339 +644,183 @@ Use case ends.
     * 2a2. User enters the command again.
     * Steps 2a1-2a2 are repeated until the data entered are correct.
     * Use case resumes from step 3.
-* 2b. HealthContact detects that the patient already exists in the database.
-    * 2b1. HealthContact shows an error message.
-    * 2b2. User enters the command again.
-    * Steps 2b1-2b2 are repeated until the patient does not exist in the database.
-    * Use case resumes from step 3.
+* 8a. HealthContact detects an error in the format of the command entered.
+    * 8a1. HealthContact shows an error message.
+    * 8a2. User enters the command again.
+    * Steps 8a1-8a2 are repeated until the data entered are correct.
+    * Use case resumes from step 9.
 
-**Use case: Adding an appointment**
+**Use case**: UC-02 - An existing patient's details are outdated<br>
```

Comment:
May change to "a patient calls to update their 'outdated' or 'recently changed' details"

Reply:
same as before